src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -291,10 +291,10 @@ private static string ConstructPathFromUri(Uri uri)
 
             if (uri != null)
             {
-                // If a path refers to a URI of form file://blah, we will convert to the local path           
+                // If a path refers to a URI of form file://blah, we will convert to the local path
```

Comment:
>path [](http://example.com/codeflow?start=95&length=4)

when you see comments without a period at the end of it, that does not meet our standard, please remember to add one. @stacywray , FOCR. do we have a code comment style guideline?

Reply:
Updated.